rust-runtime/aws-smithy-client/src/lib.rs

```diff
@@ -4,22 +4,51 @@
  */
 
 //! A Hyper-based Smithy service client.
+//!
+//! | Feature         | Description |
+//! |-----------------|-------------|
+//! | bytestream-util | ByteStream provides misuse-resistant primitives to make it easier to handle common patterns with streaming data. |
```

Comment:
From looking at the doc preview, I think these will look better with the feature name wrapped in backticks.